mdtraj/geometry/rdf.py

```diff
@@ -67,7 +70,10 @@ def compute_rdf(traj, pairs=None, r_range=None, bin_width=0.005,
         r_range = np.array([0.0, 1.0])
     r_range = ensure_type(r_range, dtype=np.float64, ndim=1, name='r_range',
                           shape=(2,), warn_on_cast=False)
-    bins = np.arange(r_range[0], r_range[1], bin_width)
+    if n_bins is not None:
```

Comment:
True, but even people that know what they're doing may make a stupid mistake -- especially if it's the kind of thing where the bin number is a computed value in their script... perhaps they made the silly mistake of doing something like

```
n_bins = lower_bound - upper_bound
```

so it comes out to the negative of what they meant.  Without a check on illegal bounds, the traceback will look like this:

```Python
>>> np.histogram(np.random.rand(10), bins=[])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../python2.7/site-packages/numpy/lib/function_base.py", line 211, in histogram
    sa.searchsorted(bins[-1], 'right')]
IndexError: index -1 is out of bounds for axis 0 with size 0
```
The simple bounds check could save even an experienced user time by showing them exactly what their mistake was.  The status quo initiates a (hopefully short) bug hunt.